pkgs/stdenv/generic/setup.sh

```diff
@@ -1508,17 +1508,7 @@ distPhase() {
 
 showPhaseHeader() {
     local phase="$1"
-    case "$phase" in
-        unpackPhase) echo "unpacking sources";;
-        patchPhase) echo "patching sources";;
-        configurePhase) echo "configuring";;
-        buildPhase) echo "building";;
-        checkPhase) echo "running tests";;
-        installPhase) echo "installing";;
-        fixupPhase) echo "post-installation fixup";;
-        installCheckPhase) echo "running install tests";;
-        *) echo "$phase";;
-    esac
+        echo "started $phase"
```

Comment:
1. I think you should explicitly print the word "phase". I am not sure which lines are printed before/after, but just "started X" might be less informative than "started phase X". Also I have tried many times to just search for "phase" in output to see what happens in which phase. By printing the word "phase", iterating through matches for "phase" would enable that.
 
```suggestion
        echo "started phase: $phase"
```

 2. I am not sure whether "started"/"finished" is the canonical terminology regarding phases. If it is, then okay, I won't complain. But I personally feel that "finished" should be used sparingly, and I also think that "begin"/"end" or "start"/"end" would be better.
 
```suggestion
        echo "begin phase: $phase"
```
```suggestion
        echo "beginning of phase: $phase"
```
```suggestion
        echo "start phase: $phase"
```
```suggestion
        echo "start of phase: $phase"
```

Reply:
@lorenzleutgeb , thank you very much for review. I agree adding `phase` word to the message is a good idea.